feathr_project/feathr/_synapse_submission.py

```diff
@@ -125,6 +125,16 @@ def get_status(self) -> str:
         assert job is not None
         return job.state
 
+    def get_job_result_uri(self) -> str:
+        """Get job output uri
+
+        Returns:
+            str: `output_path` field in the job tags
+        """
+        tags = self._api.get_spark_batch_job(self.current_job_info.tags)
```

Comment:
why is this tags?

Reply:
These are job tags (I'll update the comment to make it clear)